client/my-sites/post-type-list/index.jsx

```diff
@@ -46,6 +46,7 @@ class PostTypeList extends Component {
 		super( ...arguments );
 
 		this.renderPostRow = this.renderPostRow.bind( this );
+		this.cellRendererWrapper = this.cellRendererWrapper.bind( this );
```

Comment:
Hm, `cellRendererWrapper` needs the context... Otherwise it will throw an error when calling `this.renderPostRow`?